.travis.yml

```diff
@@ -63,6 +63,7 @@ script:
   - "python -m unittest hdf_compass.filesystem_model.test"
   - "python -m unittest hdf_compass.hdf5_model.test"
   - "python -m unittest hdf_compass.opendap_model.test"
+  - "python -m unittest hdf_compass.adios_model.test"
```

Comment:
@michaelsippel below the `conda install -q h5py lxml ...` line please add a new `pip install adios` line so we install the adios numpy wrapper in travis, too :)
(not yet packed in conda)